hardware_isolation_entry.cpp

```diff
@@ -65,6 +65,27 @@ EntrySeverity getEntrySeverityType(const openpower_guard::GardType gardType)
     }
 }
 
+openpower_guard::GardType getGuardType(const EntrySeverity severity)
+{
+    switch (severity)
+    {
+        case EntrySeverity::Critical:
+            return openpower_guard::GardType::GARD_Fatal;
+        case EntrySeverity::Manual:
+            return openpower_guard::GardType::GARD_User_Manual;
+        case EntrySeverity::Warning:
+            return openpower_guard::GardType::GARD_Predictive;
+
+        default:
+            log<level::ERR>(
+                fmt::format("Unsupported EntrySeverity [{}] "
+                            "was given to the get openpower gard type",
+                            EntryInterface::convertTypeToString(severity))
+                    .c_str());
+            return openpower_guard::GardType::GARD_Predictive;
```

Comment:
an exception should be raised

Reply:
We can not use unsupported ones from busctl or application code - it will throw compilation error but, for software
the bug we can handle.